Formula/sslsplit.rb

```diff
@@ -0,0 +1,55 @@
+class Sslsplit < Formula
+  desc "man-in-the-middle attacks against SSL encrypted network connections"
+  homepage "https://www.roe.ch/SSLsplit"
+  url "https://mirror.roe.ch/rel/sslsplit/sslsplit-0.5.0.tar.bz2"
+  sha256 "3eb13c1d0164bf04e7602d9fc45ef7460444b953efaee3ee7d52c357adb3a89a"
+  head "https://github.com/droe/sslsplit.git", :branch => "develop"
+
+  depends_on "check" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libevent"
+  depends_on "openssl"
+
+  def install
+    if ! build.head?
+      ENV.deparallelize
+      inreplace "GNUmakefile", "-o $(BINUID) -g $(BINGID)", "" # Remove UID 0,GID 0 setting for install
```

Comment:
Please use the `inreplace do` format when inreplacing the same file more than once, Thanks!